pinot-timeseries/pinot-timeseries-spi/src/main/java/org/apache/pinot/tsdb/spi/series/TimeSeriesBuilderFactory.java

```diff
@@ -25,12 +25,26 @@
 
 
 public abstract class TimeSeriesBuilderFactory {
+  protected static final int DEFAULT_MAX_UNIQUE_SERIES_PER_SERVER_LIMIT = 100_000;
```

Comment:
let's leave a TODO to make these configurable

Reply:
These are already configurable because each language developer will have to implement their own time series builder factory anyways, and the builder factory takes in a PinotConfiguration object too for initialization.

So technically users can set custom limits at tenant level if they wish to do so already.

Reply:
Ah okay makes sense!

Reply:
For your plugin you can set it to any value you'd like. You can also make it configurable by leveraging the init method in the TimeSeriesBuilderFactory class.